enshrine-esdt-safe/src/common/storage.rs

```diff
@@ -4,4 +4,7 @@ use multiversx_sc::imports::*;
 pub trait CommonStorage {
     #[storage_mapper("isSovereignChain")]
     fn is_sovereign_chain(&self) -> SingleValueMapper<bool>;
+
+    #[storage_mapper("wegldTicker")]
```

Comment:
Please use `wegldIdentifier` for storage name because it's the identifier, not the ticker